src/components/Home/Dashboard/Content/TrendingMovies/__test__/container.test.js

```diff
@@ -1,38 +1,32 @@
-/* eslint-disable no-undef */
-/* eslint-disable no-shadow */
-/* eslint-disable import/named */
-/* eslint-disable no-unused-vars */
 import React from 'react';
-import { shallow, configure } from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
-import { shallowToJson } from 'enzyme-to-json';
+import { shallow } from 'enzyme';
 import configureStore from 'redux-mock-store';
-import TopFilmsContainerConnected, { TopFilmsContainer } from '../container';
+import TrendingMoviesConnected from '../container';
 import { fetchDataRequest } from '../../../../../../store/trendingMovies/actions';
 
-configure({ adapter: new Adapter() });
-
-describe('TopFilmsContainer ', () => {
+describe('TrendingMoviesContainer ', () => {
   const store = configureStore()({
-    topFilms: {
+    trending: {
       loading: false,
-      films: [{ id: 1, title: 'Some title' }],
+      ids: [1],
       error: false,
     },
+    database: {
+      movies: { 1: { id: 1, overview: 'some text' } },
+    },
   });
   store.dispatch = jest.fn();
 
-  const wrapper = shallow(<TopFilmsContainerConnected store={store} />);
+  const wrapper = shallow(<TrendingMoviesConnected store={store} />);
   const container = wrapper.dive();
+  const instance = container.instance();
 
   it('Should match its snapshot', () => {
-    expect(shallowToJson(container)).toMatchSnapshot();
+    expect(container).toMatchSnapshot();
   });
 
-  // ПРИМЕР SPY
-  it('Check call lifeCycleMethod componentDidMount', () => {
-    mount(<TopFilmsContainerConnected store={store} />);
-    // const spy = jest.spyOn(store, 'dispatch');
+  xit('Check call lifeCycleMethod componentDidMount', () => {
```

Comment:
Enable test

Reply:
I m going to testing lifecycle and method after all.